code/modules/clothing/under/color.dm

```diff
@@ -6,7 +6,7 @@
 
 /obj/item/clothing/under/color/random/Initialize()
 	..()
-	var/obj/item/clothing/under/color/C = pick(subtypesof(/obj/item/clothing/under/color) - /obj/item/clothing/under/color/random - /obj/item/clothing/under/color/grey/glorf - /obj/item/clothing/under/color/black/ghost)
+	var/obj/item/clothing/under/color/C = pick(subtypesof(/obj/item/clothing/under/color) - subtypesof(/obj/item/clothing/under/skirt/color) - /obj/item/clothing/under/color/random - /obj/item/clothing/under/color/grey/glorf - /obj/item/clothing/under/color/black/ghost)

```

Comment:
FYI if you've typed it this way you can remove duplicate variable definitions by putting them under `clothing/under/skirt/color`

looks like this would cover `body_parts_covered`,`can_adjust`, and `fitted`

Reply:
gotcha!